vllm/engine/metrics_types.py

```diff
@@ -0,0 +1,62 @@
+import time
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import List, Optional
+
+from vllm.spec_decode.metrics import SpecDecodeWorkerMetrics
+
```

Comment:
We could add a comment here to say that these types are in a separate module to avoid having to import `vllm.engine.metrics` and therefore indirectly `prometheus_client` prematurely.

Reply:
@robertgshaw2-neuralmagic wdyt about this one? :)

Reply:
done